src/utils/validation.js

```diff
@@ -122,4 +122,32 @@ export const customValidations = {
     }
     return false;
   },
+  rageQuitMinimum({ values }) {
+    if (
+      (values.shares === '' || values.shares === 0 || values.shares === '0') &&
```

Comment:
is en empty string falsey?

wonder if we could just do something like:

```
if (!values.shares || !values.loot)
```

Reply:
Changed